src/uk/zenko/src/eu/kanade/tachiyomi/extension/uk/zenko/Zenko.kt

```diff
@@ -0,0 +1,204 @@
+package eu.kanade.tachiyomi.extension.uk.zenko
+
+import eu.kanade.tachiyomi.extension.uk.zenko.dtos.ChapterResponseItem
+import eu.kanade.tachiyomi.extension.uk.zenko.dtos.MangaDetailsResponse
+import eu.kanade.tachiyomi.extension.uk.zenko.dtos.ZenkoMangaListResponse
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimitHost
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import uy.kohesive.injekt.injectLazy
+
+class Zenko : HttpSource() {
+    override val name = "Zenko"
+    override val baseUrl = "https://zenko.online"
+    override val lang = "uk"
+    override val supportsLatest = true
+
+    override fun headersBuilder() = super.headersBuilder()
+        .add("Origin", "$baseUrl/")
```

Comment:
```suggestion
        .add("Origin", baseUrl)
```
origin header doesn't contain the trailing slash